CONTRIBUTING.md

```diff
@@ -1 +1,39 @@
\ No newline at end of file
+# Principles
+1. Whoever writes the code, choses the license
+2. Collective action, Mutual accountability
+3. Don't do or say anything that you'd want your mother to know
+
+# Why the Legal bovine excrement
+Because this interfaces financial payment system, Uncle Sam needs 
+1. Regulatory compliance with pertinent regulations [TBC title&section], now if this was just DIY it'd be an MIT license but since one of the service tiers is a hosted FaaS, someone (guess who) is going to be on the hook for leaky code
+2. Any suspicious activity of individual transaction >$10k needs to be reported to [Financial Crimes Enforcement Network](https://www.fincen.gov), goodbye lunchtime
+3. Having responsibility without legal control is like carrying a gun with safety off, do you want to take over this project?
+
+So, basically contributors transfer all ownership rights to ZTI upon acceptance of their contributions into the primary ZTI repository. Further, contributor's guarantee that they have the right to transfer ownership of their contributions; and shall indemnify ZTI from any and all repercussions caused by a violation of this guarantee. The simplest way is to understand
+* [License Zero](https://guide.licensezero.com/#comparing-public-licenses) the difference between public and private;
+```
+    def prosperity_license:
+      if commercial_user:
+        if within_trial_period:
+          return 'free to use'
+        else:
+          return 'need to buy a private license'
+      else:
+       return 'free to use'
+```
+* Reasoning behind Creative Commons [non-derivative](https://creativecommons.org/share-your-work/licensing-examples/#nd) works
+* and what open source business models [work](https://pubsonline.informs.org/doi/abs/10.1287/mnsc.1060.0547)
+
+# How to contrib meaningfully
+Eat your own dogfood. This is the standard corporate idea development
+![DevOps](./contrib/img/DevOps-BadApple.png)
+That is why the credit card processing industry looks like
+![pureed mush](./contrib/img/US-cardProcessing.png)
+
+You know that if you want to tip someone $10 that 28% is docked by the banks? Well what ZTI wants is a simple Internet Payment Interface (IPI), a microservice that will ... {do what Zee?}. In fact, you should be able to do a one-click invoice without worrying about fraudsters helping themselves to your hard earned sweat. So this project appreciates
+* security review (with enough eyes, all bugs are shallow);
+* use the testbed service to round-trip betweeen yourself and a chargeback reversion point;
+* translate into Spanish or other minority languages to allow underrepresented minorities to participate
+
+# Libre not Gratis
```

Comment:
Another great point. I wonder if this belongs more on the landing page under a title like "Why are you charging? Shouldn't open source be free?" or something

Reply:
I have a very specific order base on my years of [pyramid pitching](https://venturebeat.com/2012/09/10/an-entrepreneurs-most-powerful-tool-the-pyramid-pitch/) to investors 

1. Why?
2. How to ....
3. Cost/burden or tradeoffs (well not a financial pitch for DevOps but you get the idea)
4. Who
5. Hilights or take away points (optional)
8. Changes, now or anticipated.
9 . RTFM ... references
People TL:DR; get by with WHY and HOW. You get more differentiation at WHO so once I get text right, add internal jumppoints

Reply:
You can remember my general order with the mnemomic I came up with eye1, how2, for3, who4, hiv5 mut8 and rfc9 with zero left for indexing.